starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -52,4 +52,14 @@ public void findAll() {
     List<Inventory> actualInventory = this.inventoryDAO.findAll();
     Assert.assertFalse(actualInventory.isEmpty());
   }
+
+  // Test 'Create' DAO method
+  @Test
+  public void create() {
+
+    Inventory inventory = new Inventory();
```

Comment:
Use the inventory's setters to set `inventory`'s id, name, and product type through string constants.
View the `findAll()` test above this `create()` test to see how an object's fields are set (e.g., an inventory's id, name, and product) .